src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HPack/IntegerDecoder.cs

```diff
@@ -70,7 +73,7 @@ public bool Decode(byte b)
                 throw new HPackDecodingException(SR.net_http_hpack_bad_integer);
             }
 
-            _i = _i + ((b & 127) << _m);
+            _i = _i + ((b & 0x7f) << _m);
```

Comment:
Is this a style guide thing?

Reply:
Yeah, IMO we should be using hex for these instead of decimal across the board as these are byte operations. IDK what the rest of corefx does.